ext/mysqli/mysqli.stub.php

```diff
@@ -421,7 +421,7 @@ public function bind_result(mixed &...$vars) {}
     public function close() {}
 
     /**
-     * @return bool|null
+     * @return void
```

Comment:
Shouldn't this be `@return bool`? Also, mysqli_stmt_data_seek need changing?

Reply:
No as it only had a return false in regards to the offset position < 0, this is different from ``mysqli_data_seek()`` which returns a boolean.

Reply:
I see, I missed the mysqli_data_seek vs mysqli_stmt_data_seek distinction.